modules/installation/pages/client-requirements.adoc

```diff
@@ -7,19 +7,23 @@
 Supported operating systems for traditional and Salt clients are listed in this table.
 
 [[mgr.supported.clients]]
-[cols="1,1,1,1", options="header"]
+[cols="1,1,1,1,1", options="header"]
 .Supported Client Systems
 |===
-| Operating Systems                                    | Architecture                                 | Traditional Clients | Salt Clients
-| {sle} 11 SP4                                         | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported           | Supported
-| {sle} 12 SP3, 12 SP4                                 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported         | Supported
-| {sle} 15                                             | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported         | Supported
-|_Latest minor release {rhnminrelease6}_               | {x86}, {x86}_64                              | Supported           | Supported
-|_Latest minor release {rhnminrelease7}_               | {x86}_64                                     | Supported           | Supported
-| Open Enterprise Server 2015, 2015 SP1, 2018          | {x86}_64                                     | Supported           | Supported
-| {opensuse} Leap 15.1                                 | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 16.04                                       | {x86}_64                                     | Unsupported         | Supported
-| {ubuntu} 18.04                                       | {x86}_64                                     | Unsupported         | Supported
+| Operating System | Architecture | Traditional Clients | Salt Clients
+| {sle} 15 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported | Supported
+| {sle} 12 | {x86}_64, IBM {ppc} ({ppc64le}), {zseries}, ARM | Supported | Supported
+| {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported | Supported
+| {sle} 10 SP3+ | {x86}, {x86}_64, {ipf}, IBM {ppc}, {zseries} | Supported | Supported
+| {rhel} 8 | {x86}_64 | TBD | TBD
+| {rhel} 7 | {x86}_64 | Supported | Supported
+| {rhel} 6 | {x86}, {x86}_64 | Supported | Supported
+| CentOS 7 | {x86}, {x86}_64 | TBD | TBD
+| CentOS 6 | {x86}, {x86}_64 | TBD | TBD
+| {opensuse} Leap 15 | {x86}_64 | TBD | TBD
+| {ubuntu} 16.04 | {x86}_64 | Not supported | Supported
+| {ubuntu} 18.04 | {x86}_64 | Not supported | Supported
+| Microsoft Windows 10 | {x86}_64 | TBD | TBD
```

Comment:
No, we do not support windows

Reply:
@mcalmer We do not support it, but can't windows clients be connected via Salt or am I dreaming? :grin:

Reply:
They're on the list on  the wiki ...

Reply:
What is this list about? What salt can do or what we support in SUSE Manager ?
AFAIK the list in the WIKI comes from Don(Sales). Maybe as a Sales person he present what could be possible.

Reply:
In SUMA 4.0, Windows is definitve not `under consideration` :-)

Reply:
lol, ok taking it out!